blocks/library/block/index.js

```diff
@@ -27,8 +27,10 @@ class ReusableBlockEdit extends Component {
 		this.setTitle = this.setTitle.bind( this );
 		this.updateReusableBlock = this.updateReusableBlock.bind( this );
 
+		const { reusableBlock } = this.props;
```

Comment:
I'd thought this not to work in constructor, since `props` is otherwise passed as the first argument of the constructor function. Not sure if there's a strong consensus on whether this should be discouraged though.

Reply:
If nothing else, it's simpler! Fixed in 5af814ea52a4fc78c8398f24cdec9e4883e1d457.